app/assets/javascripts/components/app.js.jsx

```diff
@@ -0,0 +1,11 @@
+class App extends React.Component {
+  render () {
+    return (
+      <div>
+        <ul>
```

Comment:
Seems like not necessary `ul`, you not use list items in rendering `RestaurantList `